action.yml

```diff
@@ -301,6 +314,23 @@ runs:
             exit 1
           fi
 
+          # Check for deprecated code in php files and halt the merge-up if it is between majors and
+          # any deprecated code is found, unless the code as found in either Deprecation.php or DeprecationTest.php
+          if [[ $IS_MAJOR_MERGE == 1 ]]; then
+            DEPRECATED_CODE=$( \
+              git diff --cached --name-only \
+              | grep .php \
+              | grep --invert-match --extended-regexp 'Deprecation\.php|DeprecationTest\.php' \
```

Comment:
```suggestion
```
If we're adding new deprecations to the `Deprecation` class we shouldn't just let that merge up.

Reply:
Updated